_infra/helm/banner-api/Chart.yaml

```diff
@@ -4,6 +4,6 @@ description: A Helm chart for Kubernetes
 
 type: application
 
-version: 1.2.0
+version: 1.1.9
 
```

Comment:
same on this one, lets bring the app version up to 1.2.0 rather than go backwards